tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -4373,18 +4373,21 @@ void IrEmitterUnnested::EmitPrologueForReduction(
 
 void IrEmitterUnnested::EmitFullWarpShuffleDownLoopForAllReduces(
     absl::Span<HloComputation* const> reducers,
-    absl::Span<llvm::AllocaInst* const> partial_result_addresses) {
+    absl::Span<llvm::AllocaInst* const> partial_result_addresses,
+    int thread_per_block) {
```

Comment:
Sorry for nit-picking, but should be `threads_per_block`.

Reply:
Amended.
I saw some old comments that had used the singular form. I fixed them in a new commit.